Index: jessie-stage1/rootfs/boot/config.txt
===================================================================
--- jessie-stage1.orig/rootfs/boot/config.txt
+++ jessie-stage1/rootfs/boot/config.txt
@@ -54,3 +54,6 @@
 
 # Enable audio (loads snd_bcm2835)
 dtparam=audio=on
+
+start_x=1
+gpu_mem=128
Index: jessie-stage1/rootfs/etc/crontab
===================================================================
--- jessie-stage1.orig/rootfs/etc/crontab
+++ jessie-stage1/rootfs/etc/crontab
@@ -12,4 +12,5 @@ PATH=/usr/local/sbin:/usr/local/bin:/sbi
 25 6	* * *	root	test -x /usr/sbin/anacron || ( cd / && run-parts --report /etc/cron.daily )
 47 6	* * 7	root	test -x /usr/sbin/anacron || ( cd / && run-parts --report /etc/cron.weekly )
 52 6	1 * *	root	test -x /usr/sbin/anacron || ( cd / && run-parts --report /etc/cron.monthly )
+*/10 *  * * *   pi      DBX_TOKEN=$(cat /boot/timelapse_dropbox_token) /home/pi/timelapse -u $(cat /boot/timelapse_uploadname) -n $(cat /boot/timelapse_name) -s $(cat /boot/timelapse_strain) -c $(cat /boot/timelapse_controllerid) &> /var/log/timelapse.log
 #
